storage/entities/block.js

```diff
@@ -14,8 +14,343 @@
+ * @property {string} [generatorPublicKey_eql]
+ * @property {string} [generatorPublicKey_ne]
+ * @property {string} [generatorPublicKey_in]
+ * @property {string} [generatorPublicKey_like]
+ * @property {string} [payloadHash]
+ * @property {string} [payloadHash_eql]
+ * @property {string} [payloadHash_ne]
+ * @property {string} [payloadHash_in]
+ * @property {string} [payloadHash_like]
+ * @property {string} [payloadLength]
+ * @property {string} [payloadLength_eql]
+ * @property {string} [payloadLength_ne]
+ * @property {string} [payloadLength_gt]
+ * @property {string} [payloadLength_gte]
+ * @property {string} [payloadLength_lt]
+ * @property {string} [payloadLength_lte]
+ * @property {string} [payloadLength_in]
+ * @property {string} [numberOfTransactions]
+ * @property {string} [numberOfTransactions_eql]
+ * @property {string} [numberOfTransactions_ne]
+ * @property {string} [numberOfTransactions_gt]
+ * @property {string} [numberOfTransactions_gte]
+ * @property {string} [numberOfTransactions_lt]
+ * @property {string} [numberOfTransactions_lte]
+ * @property {string} [numberOfTransactions_in]
+ * @property {string} [previousBlockId]
+ * @property {string} [previousBlockId_eql]
+ * @property {string} [previousBlockId_ne]
+ * @property {string} [previousBlockId_in]
+ * @property {string} [previousBlockId_like]
+ * @property {string} [timestamp]
+ * @property {string} [timestamp_eql]
+ * @property {string} [timestamp_ne]
+ * @property {string} [timestamp_gt]
+ * @property {string} [timestamp_gte]
+ * @property {string} [timestamp_lt]
+ * @property {string} [timestamp_lte]
+ * @property {string} [timestamp_in]
+ * @property {string} [totalAmount]
+ * @property {string} [totalAmount_eql]
+ * @property {string} [totalAmount_ne]
+ * @property {string} [totalAmount_gt]
+ * @property {string} [totalAmount_gte]
+ * @property {string} [totalAmount_lt]
+ * @property {string} [totalAmount_lte]
+ * @property {string} [totalAmount_in]
+ * @property {string} [totalFee]
+ * @property {string} [totalFee_eql]
+ * @property {string} [totalFee_ne]
+ * @property {string} [totalFee_gt]
+ * @property {string} [totalFee_gte]
+ * @property {string} [totalFee_lt]
+ * @property {string} [totalFee_lte]
+ * @property {string} [totalFee_in]
+ * @property {string} [reward]
+ * @property {string} [reward_eql]
+ * @property {string} [reward_ne]
+ * @property {string} [reward_gt]
+ * @property {string} [reward_gte]
+ * @property {string} [reward_lt]
+ * @property {string} [reward_lte]
+ * @property {string} [reward_in]
+ * @property {string} [version]
+ * @property {string} [version_eql]
+ * @property {string} [version_ne]
+ * @property {string} [version_gt]
+ * @property {string} [version_gte]
+ * @property {string} [version_lt]
+ * @property {string} [version_lte]
+ * @property {string} [version_in]
+ */
+
+/**
+ * @typedef {string} fieldSets.Block
+ * @enum
+ * @value 'FIELD_SET_SIMPLE'
+ */
+
+const FIELD_SET_SIMPLE = Symbol('FIELD_SET_SIMPLE');
```

Comment:
Let's write the new "Convention" about using Symbols in DB Entities, post it into README.md and post the link to it here as a comment.

Reply:
Will be discussed and added in another PR